spec-tests/src/test/java/org/candlepin/spec/content/ContentAccessSpecTest.java

```diff
@@ -0,0 +1,860 @@
+            .contentAccessModeList("");
+
+        owner = adminClient.owners().createOwner(owner);
+
+        assertThat(owner)
+            .returns(Owners.SCA_ACCESS_MODE, OwnerDTO::getContentAccessMode)
+            .returns(Owners.ACCESS_MODE_LIST, OwnerDTO::getContentAccessModeList);
+    }
+
+    @Test
+    public void shouldLeaveModeAndListUnchangedWhenNullIsSpecified() throws Exception {
+        ApiClient adminClient = ApiClients.admin();
+        OwnerDTO owner = adminClient.owners()
+            .createOwner(Owners.random().contentAccessMode(Owners.SCA_ACCESS_MODE));
+
+        ObjectNode ownerNode = (ObjectNode) ApiClient.MAPPER.readTree(owner.toJson());
+        ObjectNode nullNode = null;
+        ownerNode.set("contentAccessModeList", nullNode);
+        ownerNode.set("contentAccessMode", nullNode);
+
+        Response response = Request.from(adminClient)
+            .setPath("/owners/{owner_key}")
+            .setPathParam("owner_key", owner.getKey())
+            .setMethod("PUT")
+            .setBody(ownerNode.toString().getBytes())
+            .execute();
+
+        OwnerDTO updatedOwner = ApiClient.MAPPER.readValue(response.getBody(), OwnerDTO.class);
+
+        assertThat(updatedOwner)
+            .returns(owner.getContentAccessMode(), OwnerDTO::getContentAccessMode)
+            .returns(owner.getContentAccessModeList(), OwnerDTO::getContentAccessModeList);
+    }
+
+    @Test
+    public void shouldNotAllowModeThatDoesNotExistInContentAccessModeList() throws Exception {
+        ApiClient adminClient = ApiClients.admin();
+        OwnerDTO owner = new OwnerDTO()
+            .id(StringUtil.random("id-"))
+            .key(StringUtil.random("key-"))
+            .displayName(StringUtil.random("display-name-"))
+            .contentAccessMode(Owners.ENTITLEMENT_ACCESS_MODE)
+            .contentAccessModeList(Owners.SCA_ACCESS_MODE);
+
+        assertBadRequest(() -> adminClient.owners().createOwner(owner));
+
+        OwnerDTO createdOwner = adminClient.owners().createOwner(Owners.random());
+
+        createdOwner.contentAccessMode("invalid-mode");
+        assertBadRequest(() -> adminClient.owners().updateOwner(createdOwner.getKey(), createdOwner));
+    }
+
+    @Test
+    public void shouldSetModeToDefaultWhenTheListIsUpdatedToNoLongerHaveTheOriginalModeValue()
+        throws Exception {
+        ApiClient adminClient = ApiClients.admin();
+        OwnerDTO scaOwner = adminClient.owners().createOwner(Owners.randomSca());
+        assertThat(scaOwner)
+            .returns(Owners.SCA_ACCESS_MODE, OwnerDTO::getContentAccessMode)
+            .returns(Owners.ACCESS_MODE_LIST, OwnerDTO::getContentAccessModeList);
+
+        // If we remove SCA mode from the list, the mode should also be forced to the default (entitlement)
+        ObjectNode objNode = ApiClient.MAPPER.createObjectNode();
+        objNode.put("contentAccessModeList", Owners.ENTITLEMENT_ACCESS_MODE);
+        Request.from(adminClient)
```

Comment:
This was interesting. In the original test ('sets mode to default when the list is updated to no longer have the original mode value') the body of the request is just `{'contentAccessModeList' : 'entitlement'}`. If I try and update the modified OwnerDTO using adminClient.owners().updateOwner(), and exception is thrown because the owner's contentAccessMode (org_environment) is no longer in the contentAccessModeList.

Reply:
You can do this via `owners().updateOwner()` but you have to set `contentAccessMode` to null to not sent it.

Reply:
Awesome! I updated the test to use `owners().updateOwner()` instead of building the Request object.